eta.c

```diff
@@ -508,6 +508,7 @@ bool calc_thread_status(struct jobs_eta *je, int force)
 				je->rate);
 		memcpy(&rate_prev_time, &now, sizeof(now));
 		for_each_rw_ddir(ddir) {
+			regrow_agg_logs();
```

Comment:
I'm ashamed to say I clearly wasn't thinking here when I wrote the original patch :-(

I'd move `regrow_agg_logs()` above the for because that function loops over all the directions anyway...

Reply:
I actually though about popping it above, agreed - fixed in 76204de